docs/how-to-guides/deploy-to-azure.md

```diff
@@ -0,0 +1,2 @@
+# Deploy to Azure
+
```

Comment:
Since there's nothing here, I guess the recommendation is: "don't"? ;)

Reply:
hahaha